oktapam/resource_ad_task_settings.go

```diff
@@ -357,7 +357,7 @@ func flattenADTaskSettings(taskSettings *client.ADTaskSettings) map[string]inter
 		m[attributes.IsActive] = *taskSettings.IsActive
 	}
 	if taskSettings.RunTest != nil {
-		m[attributes.RunTest] = *taskSettings.IsActive
+		m[attributes.RunTest] = *taskSettings.RunTest
```

Comment:
Fixing this bug, separate form this, RunTest might need more consideration as a standalone RunTest change does not make sense and is skipped anyway, but should not produce any noise with plan etc.